### DISCUSSION TITLE
Mariner file uppload

### DISCUSSION THREAD

**Discussion by Sijder** [sn4k3/UVtools#615] [Q&A] [answered]
Hey, I am running a mariner on pi zero and have tried to setup the send to option in UVtools without success so far. The url is mariner and port is 5000 which I put as is. The file uppload url is mariner:5000/#/files which I tried puting directly in host as well as in uppload files (also tried just #/files in request path). The folder is also visible to windows under Z:\ or \\mariner. Any ideas on how to make it visible to UVTools?

**Comment by sn4k3** [comment, marked as answer]
You need to use the Mapped devices on UVtools Settings instead and put the network path (samba), in your case `Z:\` or `\mariner` you just browse a folder and pick that network folder

**Reply by Sijder** [reply]
Nice thanks, it works!
